core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java

```diff
@@ -441,6 +441,13 @@ public void renameFailByDst() throws Exception {
     verifyRename(TEST_USER_1, TEST_DIR_FILE_URI, "/fileRenamed");
   }
 
+  /**
+   * verifyRename.
```

Comment:
Could you describe the method as well as the parameters? Here is an example: https://github.com/Alluxio/alluxio/blob/v1.4.0/core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java#L1109